scss/civicrm/common/_forms.scss

```diff
@@ -113,6 +113,14 @@
       @include placeholder;
       box-sizing: border-box;
       width: unset;
+
+      &.big {
+        width: 35em;
+      }
+
+      &.huge {
```

Comment:
if these are the same widths in vanilla civicrm, why do we have to define them? Are we overriding them somewhere?

Reply:
Yes, we extend `.from-control` from bootstrap which makes it  `width: 100%`.